src/main/java/edu/harvard/h2ms/web/controller/EventController.java

```diff
@@ -143,4 +141,12 @@
     return new ResponseEntity<String>(
         reportService.createEventReport(), httpHeaders, HttpStatus.OK);
   }
+
+  @PreAuthorize("hasRole('OBSERVER')")
```

Comment:
Would an admin be able to access this controller?

Reply:
I updated it for ADMIN and OBSERVER.